myapp/scheduler/stock_api_update.py

```diff
@@ -20,7 +20,6 @@ def stock_api_update():
                 'market_cap': stock['marketCap'],
                 'primary_exchange': stock['primaryExchange'],
             })
-            stock_model.save()
```

Comment:
because it's not needed, the `update_or_create` method also commits/saves the query.  you can see it here 
[https://docs.djangoproject.com/en/3.1/ref/models/querysets/#django.db.models.query.QuerySet.update](url)